### PR TITLE
286 update the visualisation logic of all charts and maps to read this value

### PR DESCRIPTION
**Pull request by ifirmawan**
- [x] Update category.json
- [x] Draw all the flows with draw.io
- [x] Update Akvo-ResponseGrouper version
- [x] Include `.category.json` to `config.json`
- [x] Modify Table
  - [x] Inspect The table logic on Frontend 
  - [x] config.js table columns
  - [x] `data/form/:form_id` api.
  - [x] Modify frontend logic for table
  - [x] Refresh views when data updated

### CODE REVIEW

File: backend/routes/chart.py [hunk 1]
```diff
@@ -74,37 +78,81 @@ def get_aggregated_pie_chart_data(req: Request,
...
+    childs = []
+    groups = groupby(data, key=lambda d: d["category"])
+    counter = defaultdict()
+    for k, values in groups:
+        for v in list(values):
+            if v["category"] in list(counter):
+                counter[v["category"]] += 1
+            else:
+                counter[v["category"]] = 1
+    for lb in labels:
```
Comment (ifirmawan) [inline backend/routes/chart.py:96]: Hi mas @dedenbangkit , I've fixed the percentages and added the correct colors and labels coming from the .category.json configuration

File: backend/db/crud_jmp.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
+from sqlalchemy.orm import Session
+from typing import List
+from AkvoResponseGrouper.views import get_categories
+from models.data import DataDict, Data
+
+# Workaround while WAI to AkvoResponseGrouper migration is in progress
+config_items = [
```
Comment (dedenbangkit) [inline backend/db/crud_jmp.py:7]: Please remove this config items @ifirmawan

File: backend/tests/test_15_1_charts.py [hunk 3]
```diff
@@ -36,28 +36,24 @@ async def test_get_aggregated_jmp_chart_data(
...
             app.url_path_for(
                 "charts:get_aggregated_jmp_chart_data",
-                form_id=1, question_id=1))
+                form_id=1, cname='1'))
         assert res.status_code == 200
         res = res.json()
         assert res == {
             "form": 1,
-            "question": 1,
+            "question": '1',
```
Comment (dedenbangkit) [inline backend/tests/test_15_1_charts.py:44]: Use string as an example instead of integer

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: backend/db/crud_jmp.py (hunk 2 above)
Comment (ifirmawan) [reply backend/db/crud_jmp.py:7]: Okay mas

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**